libs/arangodb/langchain_arangodb/chains/graph_qa/arangodb.py

```diff
@@ -429,9 +429,9 @@ def _call(
 
         chat_history = []
         if include_history and self.chat_history_store is not None:
-            for msg in self.chat_history_store.messages[-max_history_messages:]:
-                cls = HumanMessage if msg.type == "human" else AIMessage
-                chat_history.append(cls(content=msg.content))
+            chat_history.extend(
```

Comment:
good catch on not passing `role` here since we also want to include human messages